packages/containers/src/ActionBar/ActionBar.container.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+
+import { ActionBar as Component } from 'react-talend-components';
+import { getActionsProps } from '../actionAPI';
+
+function getActions(context, idOrInfo, model) {
+	if (typeof idOrInfo === 'string') {
+		return getActionsProps(context, idOrInfo, model);
+	} else if (idOrInfo.displayMode === 'splitDropdown') {
+		return Object.assign({
+			displayMode: idOrInfo.displayMode,
+			items: getActionsProps(context, idOrInfo.items, model),
+		}, getActionsProps(context, idOrInfo.name, model));
+	} else if (idOrInfo.displayMode === 'btnGroup') {
+		return {
+			displayMode: idOrInfo.displayMode,
+			actions: getActionsProps(context, idOrInfo.actions, model),
+		};
+	}
+	return undefined;
+}
+
+
+class ActionBar extends React.Component {
+	static displayName = 'CMFContainer(ActionBar)';
+	static propTypes = {
+		...Component.propTypes,
+		names: PropTypes.shape({
+			left: PropTypes.arrayOf(PropTypes.string),
+			right: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Those propTypes are not correct. You accept either a string or a splitDropdown or BtnGroup definition